earth_enterprise/src/third_party/apache2/apache-gee-httpd-ssl.conf.in.patch

```diff
@@ -0,0 +1,108 @@
+--- httpd-2.2.32.orig/docs/conf/extra/httpd-ssl.conf.in	2015-05-27 11:59:59.000000000 -0700
```

Comment:
commit https://github.com/google/earthenterprise/pull/242/commits/a8010cf5ffdf68e1088aa99f1607a27f619c93aa does not have the file.

do you get the file in src/third_party/apache2/*gee* when you apply that commit to your local branch  ?